Description: undefine 'pixel' on powerpc platforms
 gcc 4.7 on Debian/powerpc comes with the "pixel" keyword predefined:
  $ gcc -dM -E - < /dev/null | grep pixel
  #define pixel pixel
  #define __pixel __attribute__((altivec(pixel__))) unsigned short
 .
 This causes Imake to barf with error messages like:
   error: detected recursion whilst expanding macro "pixel"
Author: dann frazier <dannf@debian.org>
Bug-Debian: http://bugs.debian.org/769976
Last-Update: 2014-11-24

diff -urpN vnc4-4.1.1+X4.3.0.orig/unix/xc/config/cf/Imake.cf vnc4-4.1.1+X4.3.0/unix/xc/config/cf/Imake.cf
--- vnc4-4.1.1+X4.3.0.orig/unix/xc/config/cf/Imake.cf	2014-11-24 15:35:34.000000000 -0700
+++ vnc4-4.1.1+X4.3.0/unix/xc/config/cf/Imake.cf	2014-11-24 15:37:08.427456955 -0700
@@ -764,6 +764,7 @@ XCOMM Keep cpp from replacing path eleme
 #   define PpcArchitecture
 #  endif
 #  undef __powerpc__
+#  undef pixel /* gcc >= 4.7 defines this, causing macro recursion w/ Imake */
 # endif
 # ifdef sparc
 #  define SparcArchitecture
